platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/system/events/EventDescriptor.java

```diff
@@ -239,12 +236,16 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return new ToStringBuilder(this)
-                .append("creator", creator)
-                .append("generation", generation)
-                .append("birthRound", birthRound)
-                .append("hash", hex(hash.getValue()).substring(0, 12))
-                .toString();
+        // This intentionally does not use the ToStringBuilder
+        // This is printed frequently, and must remain concise
+        return "(id: " + creator
+                + ", gen: "
+                + generation
+                + ", br: "
+                + birthRound
+                + ", hash: "
+                + hash.toMnemonic()
+                + ")";
```

Comment:
In performance is being optimized here, perhaps `"(id: %d ...".formatted(creator, ...)` would be a good way to go. Repeated string concatenation is not particularly efficient.

Reply:
Done